.github/workflows/pages.yml

```diff
@@ -40,6 +40,9 @@ jobs:
       - name: Install react-scripts
         run: bun a react-scripts -d
 
+      - name: Ignore Typescript Compilation Errors for Build
+        run: echo 'TSC_COMPILE_ON_ERROR=true' > .env
```

Comment:
Is there a better way to do this?  Also, we might just want to turn off strict indexing here, since everything type checks besides that now.

Reply:
Seems that this flag could be moved into the `build` `script` inside `package.json` instead